packages/localstorage/README.md

```diff
@@ -1,28 +1,29 @@
-# Localstorage Service
-The `@acpaas-ui/localstorage` package allows you to easily store and retrieve data in and from your browsers storage. You can choose between different storage types and retrieve slices of the storage as observables with the `@storage` decorator. On top of that, you can easily sync your redux app state with the browser storage using the provided enhancer in the `LocalstorageStoreModule`.
+# @acpaas-ui/ngx-components/localstorage
 
-## Installation
-```
-npm install @acpaas-ui/localstorage --save
-```
+The `@acpaas-ui/ngx-components/localstorage` package allows you to easily store and retrieve data in and from your browsers storage.
 
-Import the Localstorage Module in **app.module.ts** and provide a config (optional):
+## Usage
+
+```typescript
+import { LocalstorageModule } from '@acpaas-ui/ngx-components/localstorage'`;
 ```
-import { LocalstorageModule } from '@acpaas-ui/localstorage';
 
-@NgModule({
-    imports: [
-        LocalstorageModule.forRoot({
-            storageType: 'sessionStorage',
-            identifier: 'my-app-v1,
-        })
-    ]
-})
+## Documentation
 
-export class AppModule {}
-```
+Visit our [documentation site](https://acpaas-ui.digipolis.be/) for full how-to docs and guidelines
 
-## Usage
+### Localstorage service
+
+### Methods
+
+| Method         | Description |
+| -----------    | -------------------------- |
+| `setItem(key: string, value: any): void` | store an item in the localstorage and update the subscribers (all data is stringified to a JSON string). |
+| `getItem<T = any>(key: string): T` | retrieve an item from the localstorage (all data is parsed from a JSON string). |
+| `removeItem(key: string): void` | remove an item from the localstorage and update the subscribers. |
+| `clear(...args): void` | clear the localstorage. |
```

Comment:
I would add an example of the clear method, since now it's not clear (no pun intended) how it works and what arguments I have to provide.